scs2-session-visualizer-jfx/src/main/java/us/ihmc/scs2/sessionVisualizer/jfx/session/log/VideoDataReader.java

```diff
@@ -18,7 +18,7 @@ public interface VideoDataReader
 
    void readVideoFrame(long timestamp);
 
-   void cropVideo(File file, File file2, long test, long what, ProgressConsumer heck) throws IOException;
+   void cropVideo(File outputFile, File timestampFile, long startTimestamp, long endTimestamphat, ProgressConsumer monitor) throws IOException;
```

Comment:
Hmm I kinda liked the original names :rofl: 
```suggestion
   void cropVideo(File outputFile, File timestampFile, long startTimestamp, long endTimestamp, ProgressConsumer monitor) throws IOException;
```